script.game.whatthemovie/addon.xml

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.game.whatthemovie" name="WhatTheMovie" version="1.0.9" provider-name="Tristan Fischer (sphere@dersphere.de)|tadi|skiller2k1">
+<addon id="script.game.whatthemovie" name="WhatTheMovie" version="1.0.10" provider-name="Tristan Fischer (sphere@dersphere.de)|tadi|skiller2k1">
     <requires>
         <import addon="xbmc.python" version="2.1.0"/>
         <import addon="script.module.beautifulsoup" version="3.0.8"/>
     </requires>
-    <extension point="xbmc.python.script" library="default.py" />
+    <extension point="xbmc.python.script" library="default.py">
+        <provides>game</provides>
+    </extension>
     <extension point="xbmc.addon.metadata">
         <language />
```

Comment:
@da-anda please remove language tag (it is not applied to this addon)

Reply:
It would be also nice if language files could be migrated to the gettext format (strings.po). Please also rename the commit to only contain the addon id and version:
eg "[script.x] 1.0.0"

Reply:
I'll try to fix this